Source/Blazorise/Components/Table/Table.razor.cs

```diff
@@ -350,6 +408,24 @@ public bool FixedHeader
         }
     }
 
+    /// <summary>
+    ///  Makes table have a fixed set of columns. This will make it so that the table columns could be fixed to the side of the table.
+    /// </summary>
+    [Parameter]
+    public bool FixedColumns
```

Comment:
Do we really need this? Isn't defining `Fixed` on a column enough to detect it?

Reply:
I wanted it to have it explicitly defined on a Table like this. The reason is if we do it from the TableCell up to the parent Table we would not know when to refresh the class names builder. This way it is _faster_ to render.

Reply:
Why are you so quick to say it's faster to render, what's the reason?

Why would you not know? When the `TableCell` notifies the `Table` of initialization the parameters are already set and the `Table` can then decide whether Fixed Cols should apply or not.
Also it's not like that `Table` will be able to display any fixed column at all, until one actual shows up.

Reply:
Because the parent element(Table) doesn't know how many child elements(Cells) there is. And on each cell initialize, we would have to request the parent table to refresh class, `Dirty()`.

Reply:
Does that already not happen? It still needs to find whatever column to freeze, and each time to calculate a new offset or whatever? It needs to re-render anyway?

Reply:
Not at the moment. We just calculate the sum rf the offset from the start of the row. There is no rerendering involved.